datafusion/physical-expr/src/aggregate/sum.rs

```diff
@@ -266,7 +266,7 @@ impl Accumulator for SumAccumulator {
     }
 
     fn size(&self) -> usize {
-        std::mem::size_of_val(self) - std::mem::size_of_val(&self.sum) + self.sum.size()
+        std::mem::size_of_val(self)
     }
```

Comment:
Will this size change after the AvgAccumulator /SumAccumulator struct is initialized?

Reply:
I would suggest rename the `size` method to `size_in_bytes`

Reply:
It may be changed. For example, to deal with null.

Reply:
I think not. Option is of Enum type.

Reply:
From the flamegraph, it seems it's not a bottleneck now.